plugins/paper/src/main/java/com/playmonumenta/plugins/custominventories/ClassSelectionCustomInventory.java

```diff
@@ -40,12 +40,21 @@ public class ClassSelectionCustomInventory extends CustomInventory {
 	private static final int P1_RESET_CLASS_LOC = 47;
 	private static final int P1_RESET_SPEC_LOC = 49;
 	private static final int P1_CHANGE_TRIGGERS_LOC = 51;
-	public static final ArrayList<Integer> P2_ABILITY_LOCS = new ArrayList<>(Arrays.asList(10, 14, 19, 23, 28, 32, 37, 41));
+	public static final ArrayList<Integer> P2_ABILITY_LOCS = new ArrayList<>(Arrays.asList(9, 14, 18, 23, 27, 32, 36, 41));
 	public static final ArrayList<Integer> SKILL_PAGE_SPEC_LOCS = new ArrayList<>(Arrays.asList(47, 51));
 	private static final int SKILL_PAGE_RESET_SPEC_LOC = 49;
 
 	private static final ArrayList<Integer> P3_ABILITY_LOCS = new ArrayList<>(Arrays.asList(9, 14, 18, 23, 27, 32, 36, 41));
 
+	/**
+	 *  gui identifiers are used for rp to put a texture to a gui. they would be unique to each gui and work
+	 *  pretty much like a filler.
+	 */
+	private static final int P1_GUI_IDENTIFIER_LOC = 45;
+	private static final int P2_GUI_IDENTIFIER_LOC = 45;
+	private static final int P3_GUI_IDENTIFIER_LOC = 45;
+	private static final int P4_GUI_IDENTIFIER_LOC = 45;
+
```

Comment:
Don't need 4 of these if they all identify the same location, just use GUI_IDENTIFIER_LOC for all

Reply:
These are for potentially in the future the slots are used and needed to move elsewhere - but yeah leaving these as one should work for now. Will change.